### ISSUE TITLE
Nice lib!

### ISSUE THREAD

**Issue by davestewart** [masha312/figma-variables-starter#2]
Hey,

Nice lib!

I saw your video on YouTube whilst looking to see how folks do their Tailwind stuff.

This looks like a solid start, but I''d be happy to contribute to help make the code more maintainable, and even add a few features. I'm also contributing to [FigmaToCode](https://github.com/bernaferrari/FigmaToCode/issues?q=involves%3Adavestewart+) at the moment, so I think there could be some nice tie-ins.

Are you up to speed on pull requests?

**Comment by masha312** [comment]
Hi @davestewart! Thank you and really appreciate the offer :) Would love any contributions, feature-wise and especially maintainability.

Btw, I'm currently working on some upgrades too (adding typography variables, changing up the layout) with Tyler Nickerson ([@nickersoft](https://github.com/Nickersoft)) and we will probably also refactor this to Svelte.

**Comment by davestewart** [comment]
Cool! Let me have a think and come back with some more concrete ideas.

**Comment by davestewart** [comment]
FWIW, I just forked and realised this repo might be more helpfully-named `figma-variables-starter` for both intent and visibility.

**Comment by masha312** [comment]
yess, great point! updated ✔️

**Comment by davestewart** [comment]
Hey, I jumped in last night and started simplifying. 

The main thing I wanted to solve was the duplication of the config, which was fairly straightforward, but as I understand it, the problem arises that the view would need to be bundled in order to be a single HTML file.

As such, I'd need to add a build process, then we're in the situation where it would probably just be simpler to refactor the view to with React, vue or Svelte but you've already mentioned that you're doing work to address this.

I'm mainly a Vue guy myself, though have been doing some React recently, and I have some Figma / Svelte development experience in other plugins, but I recently found this Preact lib which has already done the hard work of building out Figma-styled components:

- https://yuanqing.github.io/create-figma-plugin/ui/

I'd be happy to chuck something together (as a bit of an exploration, really) which I can PR if you like, but obviously I don't want to take away from work you're already doing if you're happy with the direction you're going in.

You also mentioned you're looking to add new properties (we have multiple sets of properties this in FigmaToCode too) and the small refactor I did on the config supports this.
